.travis.yml

```diff
@@ -4,10 +4,12 @@ node_js:
 cache:
   directories:
     - node_modules
+before_install:
+  - npm install -g npm@latest
 before_script:
   - chmod +x scripts/docs
 script:
-  - npm run cover
+  - CI=true npm run cover
```

Comment:
Thanks for the direct link to that section, not sure how I missed it. I'll check out a limited docker environment to see if I can reproduce it.